packages/sku/config/targets.json

```diff
@@ -1,4 +1,4 @@
 {
-  "browserslistNodeTarget": "node 18.20",
-  "esbuildNodeTarget": "node18.20"
+  "browserslistNodeTarget": "node 20.18.1",
+  "esbuildNodeTarget": "node20.18.1"
```

Comment:
These were intentionally only locked to minor versions I think because browserslist doesn't track anything more granular than minor versions. It also seems like [browserslist doesn't recognize anything past node 20.15](https://browsersl.ist/#q=node+%3E+20) just yet, so not sure if this is going to work 🤔

Reply:
I think this was mise auto updating this stuff.

Reply:
It's the monorepo's local postinstall script actually. https://github.com/seek-oss/sku/blob/6dab604a1be76df5d6346a000b3ac880fb829752/postinstall.js#L13

Reply:
Shall we set it to 20.15 then? I just picked the latest 20.18.1 LTS

Reply:
Yeah let's do that for now. Maybe by the time we release 20.18 will work.